tests/utils/test_authenticator.py

```diff
@@ -96,29 +96,39 @@ def test_authenticator_ignores_locked_keyring(
     http: type[httpretty.httpretty],
     with_locked_keyring: None,
     caplog: LogCaptureFixture,
+    mocker: MockerFixture,
 ) -> None:
     caplog.set_level(logging.DEBUG, logger="poetry.utils.password_manager")
+    spy_get_credential = mocker.spy(PoetryKeyring, "get_credential")
+    spy_get_password = mocker.spy(PoetryKeyring, "get_password")
     authenticator = Authenticator()
     authenticator.request("get", "https://foo.bar/files/foo-0.1.0.tar.gz")
 
     request = http.last_request()
     assert request.headers["Authorization"] is None
-    assert "Keyring foo.bar is locked" in caplog.messages
```

Comment:
Shouldn't there still be a message that the keyring is locked in the output?

Reply:
Yeah "Accessing keyring failed during availability check" will add.